cockroachcloud/architecture.md

```diff
@@ -0,0 +1,76 @@
+---
+title: CockroachCloud Architecture
+summary: Learn more about CockroachCloud architecture
+toc: true
+---
+
+CockroachCloud is a fully-managed deployment of CockroachDB. This page describes CockroachCloud's architecture and how it relates to CockroachDB.
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="serverless">CockroachCloud Serverless </button>
+  <button class="filter-button" data-scope="dedicated">CockroachCloud Dedicated</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+## CockroachCloud Serverless
+
+CockroachCloud Serverless (beta) is a fully-managed, multi-tenant deployment of CockroachDB. Being familiar with the following concepts will help you understand what our Serverless architecture achieves.
```

Comment:
Do we have to emphasize multi-tenant here? That's kind of a negative. I'd say instead that it's a fully-managed, auto-scaling deployment.